tests/oss-fuzz/Makefile.oss-fuzz

```diff
@@ -3,8 +3,6 @@
 top_builddir=../..
 top_srcdir=../..
 
-LIBS=-lFuzzingEngine
```

Comment:
(See above: This should be LDLIBS). It should get passed to Makefile.ci automatically.